### ISSUE TITLE
Facebook Logo doesn't comply with current branding guidelines

### ISSUE THREAD

**Issue by greenweeds** [Automattic/jetpack#15954]
The default icons used for Facebook in social sharing don't comply with current branding guidelines. A screensnip of the out of date icon being served is attached [1], along with the preferred icon [2]. 

Up to date branding can be accessed at https://en.facebookbrand.com/ 

[1] 
<img width="474" alt="default-facebook-icon-in-jetpack" src="https://user-images.githubusercontent.com/3043487/83043579-f4925700-a03a-11ea-8201-a2ddd97e6915.png">

[2] 
![f_logo_RGB-Blue_72](https://user-images.githubusercontent.com/3043487/83043935-68346400-a03b-11ea-9fca-a1ec5712ef24.png)

**Comment by jeherve** [comment]
For reference, the logo comes from here:
https://github.com/Automattic/social-logos

Once it is updated in that repo, we can update our copy of social logos here in Jetpack, and adjust the CSS of the sharing buttons to make an exception for the Facebook icon so it can come until the edge of the circle vs. being in the center like all the others today.

We'll also need to update the SVG and the CSS used in the Social Menus and Social Icons widget:
https://github.com/Automattic/jetpack/blob/63fee0f63b8cb1e7909d8b98a50519f9a12eb800/modules/theme-tools/social-menu/social-menu.svg

**Comment by keoshi** [comment]
Thanks for bringing this up, @greenweeds !

Yeah, it seems FB's guidelines have become really restrictive in terms of the logo options they offer (which is now just one).

Ditto on what @jeherve said above on creating an exception for FB, even though the circle should ideally be part of the logo. It should look like this in relation to other logos:

![image](https://user-images.githubusercontent.com/390760/83140023-48f21100-a0e5-11ea-95b8-1c3e4a95e897.png)

This brings up the question of revising all social logos and their latest guidelines and, eventually and more importantly, decide if we still offer multiple options in terms of social icon styling for this block.

**Comment by folletto** [comment]
I think the quick fix for this is:

![Screenshot 2020-05-28 at 13 18 23](https://user-images.githubusercontent.com/4389/83140331-d170b180-a0e5-11ea-947c-f7c6fce96fd8.png)

Mostly because of what you mentioned it would create and exception for Facebook, and it would also not work with the other styles for social sharing.

**Comment by keoshi** [comment]
Great point, @folletto — that would allow us to stay in line with their guidelines while still offering various styling flavors.

**Comment by jeherve** [comment]
I'm adding the Needs Design label on this one again, as I am not quite sure how to go about updating the Social Logos repo with an updated SVG and icon in the font family, that would reflect the design changes above. @keoshi Do you think you could take a look at this?

Thank you!

**Comment by folletto** [comment]
> I am not quite sure how to go about updating the Social Logos repo with an updated SVG and icon in the font family, that would reflect the design changes above

The logo seems already updated there: https://github.com/Automattic/social-logos/pull/94

You probably just need to pull the latest version?

**Comment by jeherve** [comment]
Reopening as there are still a few places where we'd need to update things:

- Publicize in wp-admin

![image](https://user-images.githubusercontent.com/426388/89293507-4fa98100-d65e-11ea-92e0-be1b70c42b42.png)

- Publicize in Calypso

![image](https://user-images.githubusercontent.com/426388/89293630-81224c80-d65e-11ea-9d37-e40a7b3abd56.png)

- The comment form on WordPress.com and Jetpack

![image](https://user-images.githubusercontent.com/426388/89293660-8da6a500-d65e-11ea-859c-7f0920976749.png)

**Comment by mdrovdahl** [comment]
Noting a related issue...when using the Jetpack Social Media Icons Widget (https://jetpack.com/support/extra-sidebar-widgets/social-icons-widget/) AND the Jetpack Social Sharing (https://jetpack.com/support/sharing/) set to "icon only" we end up with two different Facebook icons...

![ephemeral-mdrovdahl-20200907 atomicsites blog_2020_09_07_hello-world_](https://user-images.githubusercontent.com/474467/92405780-35dedc00-f0eb-11ea-99dc-870ec215369d.png)

**Comment by jeherve** [comment]
@mdrovdahl Are you referring to the fact that the sharing button has the logo inside a circle? If so, that's something we discussed here:
https://github.com/Automattic/jetpack/issues/15954#issuecomment-635304558

**Comment by mdrovdahl** [comment]
@jeherve I think...yes? I read the comments but I'm not clear if the screenshot I provided is indeed the expected behavior. It seems odd that Jetpack places two different Facebook icons on that page (the first a blue circle with a white circle inside with a white f on the inside and the second a red circle with a white f on the inside).

**Comment by jeherve** [comment]
That is indeed the expected behaviour, due to the limitations of the sharing button design.

**Comment by Robertght** [comment]
I can see we had this added to TOB, but since this so old, I run some tests and this is how I see the icon now:

Marketing > Connections:

![W4hrOMRNixz3hvqT0ft1iPeW4bvFiu2YsCJY34qj.jpg](https://github.com/user-attachments/assets/44f8cbe8-4cc4-4da8-9166-df0bb72238c7)

Twentyninteen's social menu:

![Image](https://github.com/user-attachments/assets/36652636-d53a-40c7-86db-c150ed2666a9)

Social icons widget with 2019 and social icons block with 2024:

![Image](https://github.com/user-attachments/assets/ec112f1a-b7b4-44b8-a698-a42f6492fd55)

@jeherve is this still relevant?

**Comment by jeherve** [comment]
Yes, it's still relevant. The logo's color has been updated, but not its shape, as discussed above:
https://github.com/Automattic/jetpack/issues/15954#issuecomment-635300488
